### PR TITLE
Issue #8

### PR DESCRIPTION
**Pull request by uptownhr**
Added breakpoints for mobile

- [ ] Waiting on SVG for arrow

### CODE REVIEW

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 1]
```diff
@@ -54,69 +54,77 @@ export default {
 
 <style lang="scss">
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:55]: Your styles should be scoped here

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 2]
```diff
@@ -54,69 +54,77 @@ export default {
...
 .banner-highlight {
+    color: var(--color-company);
     display: block;
     position: relative;
     height: 80vh;
+}
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:61]: All the other styles should be nested inside this block.

Like:

```
.banner-highlight {
     .block-text {}
     .text {}

      // Breakpoints
}
```

File: src/styles/variables.scss [hunk 3]
```diff
@@ -6,4 +6,5 @@
 // Breakpoints
 $gt-wide: "only screen and (min-width: 1800px)";
 $lt-tablet: "only screen and (max-width: 1024px)";
+$lt-phone: "only screen and (max-width: 414px)";
```
Comment (drewbaker) [inline src/styles/variables.scss:9]: This is not needed for this component, but also below 1024 you can use all the same styles on this site. Meaning, we go to phone view below 1024.

Also, pro tip, an iPhone is actually 750px wide (it displays like it's 375px, but double density makes CSS measure it at 750px). No device in 2020 would ever be 414px wide.

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 4]
```diff
@@ -54,69 +54,77 @@ export default {
...
-    // @media #{$has-hover} {
-    //     &:hover {
-    //     }
-    // }
-    // Breakpoints
-    // @media #{$lt-phone} {
-    // }
+    .block-text {
+        padding: 2rem 1rem;
+    }
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:128]: breakpoints are not needed on this component.

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 5]
```diff
@@ -54,69 +54,77 @@ export default {
...
+
+.prompt-container {
+    position: absolute;
+
+    .prompt {
+        font-size: 28px;
+        overflow: hidden;
+        transition: opacity 200ms ease;
+        opacity: 0;
+        padding: 0 1rem 0 0;
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:93]: This should be 15px. We don't use rem's.

File: src/components/BannerHighlight/BannerHighlight.vue [hunk 6]
```diff
@@ -54,69 +54,77 @@ export default {
...
-        }
+    &:hover .prompt {
+        opacity: 1;
+    }
 
-        &:hover .prompt {
-            opacity: 1;
-        }
+    &:hover .arrow {
+        left: 100%;
```
Comment (drewbaker) [inline src/components/BannerHighlight/BannerHighlight.vue:112]: This should be inside a $has-hover media query, also animate the translate, not left. In general, animating transforms is way smoother than a left/right/top etc.

**Comment by uptownhr** [comment]
@drewbaker again, I accidentally merged this in but this time figured out what's causing it. Because i'm also a contributor on the main factory repo, github is automatically merging this in for me. Going to adjust my settings so this doesn't happen in the future.

**Pull request merged**